README.md

```diff
@@ -103,7 +103,7 @@ releases of the toolkit.
 
 You can download and run pre-built versions of GATK4 from the following places:
 
-* A zip archive with everything you need to run GATK4 can be downloaded for each release from the [github releases page](https://github.com/broadinstitute/gatk/releases).
+* A zip archive with everything you need to run GATK4 can be downloaded for each release from the [github releases page](https://github.com/broadinstitute/gatk/releases). We also host unstable archives generated nightly in the Google bucket gs://gatk-nightly-builds.
```

Comment:
Do nightly builds ever age away or is dockerhub hosting all of our builds for all eternity?

Reply:
They just sit forever I believe. Its smart about compressing them/reusing layers but it doesn't seem like dockerhubs pricing scheme cares about the number of images hosted compared the the number of repositories but I could be wrong about that.